src/common/Utilities/Util.h

```diff
@@ -713,54 +713,44 @@ class EventMap
         ScheduleEvent(eventId, time, groupId, phase);
     }
 
-        /**
-        * @name RescheduleEvent
-        * @brief Cancels the given event and reschedules it.
-        * @param eventId The id of the event.
-        * @param time The time in milliseconds until the event occurs.
-        * @param group The group which the event is associated to. Has to be between 1 and 8. 0 means it has no group.
-        * @param phase The phase in which the event can occur. Has to be between 1 and 8. 0 means it can occur in all phases.
-        */
-        void RepeatEvent(uint32 time)
+    /**
+    * @name RescheduleEvent
+    * @brief Cancels the given event and reschedules it.
+    * @param eventId The id of the event.
+    * @param time The time in milliseconds until the event occurs.
+    * @param group The group which the event is associated to. Has to be between 1 and 8. 0 means it has no group.
+    * @param phase The phase in which the event can occur. Has to be between 1 and 8. 0 means it can occur in all phases.
+    */
+    void RepeatEvent(uint32 time)
+    {
+        _eventMap.insert(EventStore::value_type(_time + time, _lastEvent));
+    }
+
+        if (itr->first > _time)
+            return 0;
+        else if (_phase && (itr->second & 0xFF000000) && !((itr->second >> 24) & _phase))
```

Comment:
add the curly braces here pls
moreover, you could use a constant for `0xFF000000`, I saw that it is used more times.

Reply:
its kinda... no offense but this file got effected by astyle commit which triggered mass of conflicts, when conflict was resolved, NOW this gets into inspection for style? Seriously why didn't used Astyle to add auto brackets...

Reply:
Resolved